projects/m1/024-fizz-buzz/js/index.js

```diff
@@ -0,0 +1,28 @@
+function playFizzBuzz(){
+    let phrases = '';
+
+    const length = 100;
+
+    for (let i = 1; i <= length; i++) {
+        let phrase = i;
+
+        if ( i % 3 === 0 ){
+            phrase = 'fizz';
+        } else if ( i % 5 === 0){
+            phrase = 'buzz';
+        } else if ( i % 3 === 0 && i % 5 === 0){
+            phrase = 'fizz and buzz';
+
+        }
+
+        phrases += i === length ? phrase : phrase + '\n';

```

Comment:
This check is also not recommended.
It makes **no sense to check 100 values ​​to remove the trailing '\n'**.
Find a workaround to remove it ;-)